core/controllers/release_coordinator.py

```diff
@@ -111,6 +112,12 @@ class FeatureFlagsHandler(
                 },
                 'default_value': None
             },
+            'default_value': {
+                'schema': {
+                    'type': 'bool'
+                },
+                'default_value': None
+            },
```

Comment:
Also, here you are defining `default_value` as a `bool` type but in your `TypedDict` you are defining it as a mixed type( Union type). Please change the type there (if you are sure that the type is only going to be `bool`).

Reply:
Thank you!
Its resolved now.